src/commands/staff/ParseRaidVc.ts

```diff
@@ -0,0 +1,110 @@
+import {BaseCommand, ICommandContext, ICommandInfo} from "../BaseCommand";
```

Comment:
Works but could be a little prettier
![image](https://user-images.githubusercontent.com/97406520/149035811-01dfbbd2-bc49-4436-994c-c0ff8e61706d.png)